src/main/java/com/xiaomi/infra/pegasus/client/PegasusClient.java

```diff
@@ -477,6 +478,27 @@ public long incr(String tableName, byte[] hashKey, byte[] sortKey, long incremen
     return tb.incr(hashKey, sortKey, increment, 0);
   }
 
+  @Override
+  public <Req, Res> void batch(String tableName, Batch<Req, Res> batch) throws PException {
+    PegasusTable tb = getTable(tableName);
+    tb.batch(batch, 0);
+  }
+
+  @Override
+  public <Req, Res> void batch(String tableName, Batch<Req, Res> batch, List<Res> responses)
+      throws PException {
+    PegasusTable tb = getTable(tableName);
+    tb.batch(batch, responses, 0);
+  }
+
+  @Override
+  public <Req, Res> void batchWaitAllComplete(
+      String tableName, Batch<Req, Res> batch, List<Pair<PException, Res>> responses)
+      throws PException {
+    PegasusTable tb = getTable(tableName);
+    tb.batchWaitAllComplete(batch, responses, 0);
+  }
+
```

Comment:
We gonna deprecate the client interface. This is not the best practice we encourage. So leave PegsausClientInterface unchanged.